indigo_social/models.py

```diff
@@ -13,10 +36,16 @@ class UserProfile(models.Model):
     organisations = models.TextField(blank=True, null=True, default='', help_text="Organisation(s)")
     specialisations = models.TextField(blank=True, null=True, default='', help_text="Specialisation(s)")
     areas_of_law = models.CharField(max_length=256, blank=True, null=True, default='', help_text="Area(s) of law")
-    profile_photo = models.ImageField(blank=True, null=True)
+    profile_photo = models.ImageField(upload_to=user_profile_photo_path, blank=True, null=True)
     twitter_username = models.CharField(max_length=256, blank=True, null=True, default='')
     linkedin_profile = models.URLField(blank=True, null=True, default='')
 
+    @property
+    def profile_photo_url(self):
+        if not self.profile_photo:
+            return "/static/images/avatars/default_avatar.svg"
+        return self.profile_photo
```

Comment:
This is where we'd return something like `reverse('user_profile_photo', kwargs={'username': self.user.username, 'nonce': self.profile_photo_nonce}` which would be a url such as `/contributors/greg/avatar/abc123`